static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -47,6 +47,8 @@ define([
                     }).join('&');
                 }).join('&');
         },
+        /*
+        To reinstate when find out where required
```

Comment:
actually I really just thought it was funny! but actually on reflection it is a good comment cos it explains what you were *thinking* when you made the change